Gems/ROS2/Code/Include/ROS2/GNSS/GNSSPostProcessingRequestBus.h

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+#pragma once
+
+#include "AzCore/EBus/EBusSharedDispatchTraits.h"
+#include <AzCore/Component/EntityId.h>
+#include <AzCore/Interface/Interface.h>
+#include <AzCore/std/string/string.h>
+#include <sensor_msgs/msg/nav_sat_fix.hpp>
+
+namespace ROS2
+{
+    //! Interface class that allows to add post-processing to the pipeline
+    //!
+    //! Each function call can be processed without blocking Bus for other dispatches.
+    //! Do not use connects / disconnects to this bus during event dispatch, as they are not allowed for this concurrency model.
+    //! Those constraints allow for processing multiple gnss frames in the same time.
```

Comment:
Please keep `GNSS` capitalized.
```suggestion
    //! Those constraints allow for processing multiple GNSS frames at the same time.
```